test/files/run/range.scala

```diff
@@ -37,15 +37,17 @@ object Test {
     def gr1 = NumericRange(x, x, x)
     def gr2 = NumericRange.inclusive(x, x, x)
     def gr3 = NumericRange(x, x * fromInt(10), x)
-    def gr4 = NumericRange.inclusive(x, x * fromInt(10), x)
-    def gr5 = gr3.toList ::: negated.gr3.toList
+    // SI-9348
```

Comment:
Please move the comment to the end of the next line, to avoid the confusion that all of the subsequent methods relate to SI-9348.